Sources/iPhoneNumberField/iPhoneNumberField.swift

```diff
@@ -179,7 +179,7 @@ public struct iPhoneNumberField: UIViewRepresentable {
         } else {
             uiView.withExamplePlaceholder = autofillPrefix
         }
-        if autofillPrefix && displayedText.isEmpty && isFirstResponder { uiView.resignFirstResponder() } // Workaround touch autofill issue
```

Comment:
Instead of deleting this line shouldn't we do this?

```swift
if autofillPrefix && displayedText.isEmpty && isFirstResponder { // Workaround touch autofill issue
  DispatchQueue.main.async {
    uiView.resignFirstResponder()
  }
}
```